uber/templates/summary/shirt_counts.html

```diff
@@ -21,4 +21,19 @@
     </table>
 {% endfor %}
 
+<h3>Shirt Sales by Week</h3>
```

Comment:
I moved this out of ``summary/index`` to this page, since ``summary/index`` is a more commonly loaded page and most people looking at it don't care about these counts, which take long enough to compute that it made more sense to move it here.